support/src/components/knowledge-base/PublicKBArticle.js

```diff
@@ -0,0 +1,11 @@
+import TextEditor from "../TextEditor";
+
+const PublicKBArticle = ({ item }) => {
```

Comment:
[Please IGNORE ME!] Là c'est un peu tard, mais j'ai souvent vu les équipes chercher à éviter les acronymes (sauf exception bien sûr, genre `props`), en tout cas c'est ce qu'on faisait chez ForestAdmin et c'était assez aprécié.

```suggestion
const PublicKnowledgeBaseArticle = ({ item }) => {
```

Reply:
je suis daccord !
il est encore temps de le changer je pense (a moins que ce soit vraiment trop relou @arnaudambro )

mais plus on est explicite, mieux c'est je pense. mais idem, tu peux nous envoyer chier si tu veux pas ahah, `KB` fera l'affaire (on a deja mis des trucs pires dans le snu)